### DISCUSSION TITLE
bower ENOENT No bower.json present

### DISCUSSION THREAD

**Discussion by almleoalmeida** [frontendbr/forum#1189] [Perguntas e Respostas]
Falaê

Estava instalando o MEAN na minha máquina e não consegui executar o comando **bower install** pois ele apresenta o erro **bower ENOENT No bower.json present**. A recomendação é que eu migre para o Yarn, no entanto não sei fazer isso.

**Comment by felipefialho** [comment]
Na estrutura desse projeto tem o arquivo `bower.json`?

**Comment by almleoalmeida** [comment]
tem sim

**Comment by felipefialho** [comment]
Bower está instalado?
